cmd/run-function/runcmd.go

```diff
@@ -0,0 +1,217 @@
+package run_function
```

Comment:
Annoying but important nitpick. I think the package should be `runfn` like the command, and the file should be `runfn.go`.

Reply:
done!